test/erlcloud_sns_tests.erl

```diff
@@ -168,7 +169,12 @@ output_test(Fun, {Line, {Description, Response, Result}}) ->
       fun() ->
               meck:expect(erlcloud_httpc, request, output_expect(Response)),
               erlcloud_ec2:configure(string:copies("A", 20), string:copies("a", 40)),
-              Actual = Fun(),
+              Actual = try
+                  Fun()
+                catch
+                  _Class:Error:_Stack ->
```

Comment:
Technically no need to match `:_Stack` here, it's just wasted characters.
```suggestion
                  _Class:Error ->
```

Reply:
Removed stacktrace matching